README.md

```diff
@@ -6,40 +6,28 @@ The server components for the new AcousticBrainz project.
 Please report issues here: http://tickets.musicbrainz.org/browse/AB
 
 
-## Installation
+## Installation and Running
 
 ### Vagrant VM
 
-The easiest way to start is to setup ready-to-use [Vagrant](https://www.vagrantup.com/)
-VM. To do that [download](https://www.vagrantup.com/downloads.html) and install
-Vagrant for your OS. Then copy two config files:
+You can use [docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/) to run the AcousticBrainz server. Make sure docker and docker-compose are installed.
+Then copy two config files:
 
 1. `custom_config.py.example` to `custom_config.py` *(you don't need to modify this file)*
 2. `profile.conf.in.sample` to `profile.conf.in` in the `./hl_extractor/` directory
 *(in this file you need to set `models_essentia_git_sha` value)*
 
-After that you can spin up the VM and start working with it:
+Then, in order to download all the software and build and start the containers needed to run AcousticBrainz, use the following command:
 
-    $ vagrant up
-    $ vagrant ssh
+`docker-compose -f docker/docker-compose.dev.yml up --build`
 
-There are some environment variables that you can set to affect the
-provisioning of the virtual machine.
+The first time you do that, you will then need to initialize the AcousticBrainz database:
 
- * `AB_NCPUS`: Number of CPUs to put in the VM (default 1, 2 makes
-               compilation faster)
- * `AB_MEM`:   Amount of memory (default 1024mb)
- * `AB_MIRROR`: ubuntu mirror (default archive.ubuntu.com)
- * `AB_NOHL`: If set, don't compile the highlevel calculation tools
-              (not needed for regular server development)
+`docker-compose -f docker/docker-compose.dev.yml run webserver python2 manage.py init_db --skip-create-db`
 
+In order to load a psql session, use the following command:
 
-There are some shortcuts defined using fabric to perform commonly used
-commands:
-
- * `fab vpsql`: Load a psql session. Requires a local psql client
- * `fab vssh`: Connect to the VM more efficiently, saving the settings
-               so that you don't need to run vagrant each time you ssh.
+`docker-compose -f docker/docker-compose.dev.yml run webserver psql -U acousticbrainz -h db`
```

Comment:
just thinking out loud about this - do we need the psql client in the web container? It's in the db container, so we can still have a variation of this line, but let's not install things that aren't needed in the web container